css/style.css

```diff
@@ -0,0 +1,100 @@
+body {
+  background-color: #f4f5f6;
+}
+
+@media only screen and (min-width: 768px) {
+  #md-nav {
+    position: absolute;
+    left: -999999px;
+  }
+}
+
+@media only screen and (max-width: 768px) {
+  #lg-nav {
+    position: absolute;
+    left: -999999px;
+  }
+}
+
+.page-header .logo {
+  display: block;
+  width: 132px;
+  height: 20px;
+  fill: white;
+}
+
+.page-header .big-logo {
+  display: block;
+  width: 350px;
+  height: 90px;
+  padding: 20px;
+  fill: white;
+}
+
+header nav {
+  background-color: #ff0500;
+}
+
+header nav .trapezoid {
+  border-bottom: 43px solid white;
+  border-right: 25px solid transparent;
+  height: 0;
+  width: 25px;
+}
+
+.border-top {
+  border-bottom: 1px solid #dee2e6 !important;
+  margin-top: 20px;
+}
+
+@media only screen and (min-width: 992px) {
+  header nav .main-links-lg {
+    width: 93%;
+    position: absolute;
+    bottom: -15px;
+  }
+}
+
+.d-lg-block {
+  display: block !important;
+}
+
+.border,
+.about-us ul li,
+.editions {
+  border-right: 0.5px solid #d9d9d9 !important;
+  height: 50%;
+}
+
+.icon-input {
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+}
```

Comment:
- [ ] This can be done with the bootstrap classes `d-flex` and `align-items-center`